app.js

```diff
@@ -4,7 +4,7 @@ const fs = require("fs");
 const bodyParser = require("body-parser");
 const logger = require("./utils/logger");
 
-require("dotenv").config({ path: __dirname + "/.env" });
+require("dotenv").config({path: __dirname + "/.env.sample" });
```

Comment:
all the environment variables are stored in .env only. .env.sample just tells the names of the variables, but there actual values are stored and taken from .env only.